modules/@apostrophecms/image-widget/index.js

```diff
@@ -7,18 +7,32 @@ module.exports = {
     dimensionAttrs: false,
     placeholder: true,
     placeholderClass: false,
-    placeholderImage: 'jpg'
+    placeholderImage: 'jpg',
+    // 0 means disabled width setting
+    defaultImageWidth: 0,
+    imageResizeStep: 5
   },
-  fields: {
-    add: {
-      _image: {
-        type: 'relationship',
-        label: 'apostrophe:image',
-        max: 1,
-        required: true,
-        withType: '@apostrophecms/image'
+  fields(self) {
+    return {
+      add: {
+        _image: {
+          type: 'relationship',
+          label: 'apostrophe:image',
+          max: 1,
+          required: true,
+          withType: '@apostrophecms/image'
+        },
+        width: {
+          type: 'range',
+          label: 'apostrophe:imageWidth',
+          help: 'apostrophe:imageWidthHelp',
+          min: 0,
+          max: 100,
+          step: self.options.imageResizeStep || 1,
+          def: self.options.defaultImageWidth ?? 100
```

Comment:
This is going to work out to 0, not 100, because 0 is not nullish. So I think the `?? 100` does not do anything useful.

Now, assuming we just remove `?? 100`...

The user is going to see a slider set to 0, but a rendering that is probably 100% (whatever the CSS does by default).

But, this does result in backwards compatible behavior by default, and even provides a way to return to default behavior.

And "0%" itself is not a useful setting, therefore it's OK to customize the behavior.

So I guess I like it.